packages/admin-ui/src/search/search.i18n.ts

```diff
@@ -0,0 +1,12 @@
+export const messages = {
+  'en-US': { search: 'Search', clearButton: 'Clear search input' },
```

Comment:
Is there an issue with the localization team for this file? How is this going to scale in the future?